sdk/security/keyvault/internal/fake_challenge.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+package internal
```

Comment:
```suggestion
// Licensed under the MIT License. See License.txt in the project root for license information.

package internal
```
a comment adjacent to `package` is documentation of the package i.e., it ends up on pkgsite

Reply:
fixed in 063183e